CHANGES.md

```diff
@@ -80,6 +80,8 @@ To be released.
     a high latency internet connection.  [[#709]]
  -  `Swarm<T>.BootstrapAsync()` became to report `PeerDiscoveryException`
     instead of `SwarmException` directly. [[#604], [#726]]
+ -  `BlockChain<T>.Append()` became to unstage the staged `Transaction`s which
+    nonce was already stored in chain by same signer.  [[#721], [#728]]
```

Comment:
How about this?

> `BlockChain<T>.Append()` became to unstage the staged `Transaction<T>`s that have lower nonce than the highest nonce of the same signer's transactions in the same chain, since these virtually never become valid.